app/src/main/java/com/nononsenseapps/feeder/ui/compose/feed/FeedScreen.kt

```diff
@@ -495,145 +509,393 @@ fun FeedScreen(
         onDelete = onDeleteFeeds,
         onEditFeed = onEditFeed,
         toolbarActions = {
-            if (viewState.currentFeedOrTag.isNotSavedArticles) {
-                PlainTooltipBox(
-                    tooltip = {
-                        Text(stringResource(id = R.string.filter_noun))
+            if (viewState.searchBarVisible) {
+                SearchBar(
+                    expanded = true,
+                    onExpandedChange = { onShowSearchBar(it) },
+                    inputField = {
+                        SearchBarDefaults.InputField(
+                            query = viewState.filter.search,
+                            onQueryChange = { filterCallback.searchFor(it) },
+                            onSearch = {
+                                onShowSearchBar(false)
+                                filterCallback.searchFor(it)
+                            },
+                            expanded = true,
+                            onExpandedChange = { onShowSearchBar(it) },
+                            placeholder = { Text("query") },
+                            leadingIcon = { Icon(Icons.Default.Search, contentDescription = "search") },
```

Comment:
contentDescriptions need translated string as well.

It should also describe the image. In this case "search icon" would be more appropriate, or maybe even "null" (there is no point in including the icon in a screen reader because it is purely decorative)